api/v1/db.go

```diff
@@ -86,22 +86,34 @@ func UpdateMeta(c *gin.Context) {
 			return
 		}
 		logger.LogDebug(meta)
-		meta.AttrMap, err = ConvertDynamoToMap(meta.TableName, meta.DynamoObject)
+		meta.AttrMap, err = ConvertDynamoToMap(meta.TableName, meta.Item)
 		if err != nil {
 			c.JSON(errors.New("ValidationException", err).HTTPResponse(meta))
 			return
 		}
-		meta.ExpressionAttributeValues, err = ConvertDynamoToMap(meta.TableName, meta.DynamoObjectAttr)
+		meta.ExpressionAttributeMap, err = ConvertDynamoToMap(meta.TableName, meta.ExpressionAttributeValues)
 		if err != nil {
 			c.JSON(errors.New("ValidationException", err).HTTPResponse(meta))
 			return
 		}
 
-		res, err := put(c.Request.Context(), meta.TableName, meta.AttrMap, nil, meta.ConditionalExp, meta.ExpressionAttributeValues)
+		for k, v := range meta.ExpressionAttributeNames {
+			meta.ConditionExpression = strings.ReplaceAll(meta.ConditionExpression, k, v)
```

Comment:
Why this is needed here?

Reply:
if changed inside services.Put() then other function calls need to be changed